src/components/ChattingScreen.js

```diff
@@ -1,31 +1,127 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
 import Header from './Header';
 import MessageSender from './MessageSender';
 
 export default function ChattingScreen() {
-	const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
-	const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
+	const liLeftside = {	// 고은님 메시지 li
+		display: 'flex',
+		alignItems: 'center',
+		padding: '15px 0'
+	};
+	const liRightside = {	// 정쿨님 메시지 li
+		display: 'flex',
+		justifyContent: 'flex-end',
+		alignItems: 'center',
+		padding: '15px 5px'
+	};
+	const EunkoImageStyle = {	// 고은님 메시지 image
+		height: '50px',
+		paddingTop: '0%',
+		borderRadius: '40%',
+		boxShadow: '3px 1px 7px  rgba(0, 0, 0, 0.25)'
+	};
+	const CoolImageStyle = {	// 정쿨님 메시지 image
+		height: '50px',
+		paddingTop: '0%',
+		borderRadius: '40%',
+		boxShadow: '3px 1px 7px  rgba(0, 0, 0, 0.25)'
 
-	const MSGLIST = [
+	};
+
+	// 입력받은 메시지 저장소
+	const [messageArr, setMessageArr] = useState( [
 		{ user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
-		{ user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
-		{ user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
+		{ user: true, content: '여러분께 소개드리고 싶은 사람이 있어요~'},
+		{ user: true, content: '바로 12기의 장창훈 이라고 하시는 분입니다 !!!' },
 		{ user: false, content: '안녕 은아' },
 		{ user: false, content: '뭐해 ?' },
-		{ user: true, content: '시원아 넌 최고의 팀장이야' },
+		{ user: true, content: '지금 장창훈님을 소개하고있어' },
 		{ user: false, content: '나도 아니까  ' },
 		{ user: false, content: '그만 말해줘도 돼' },
-		{ user: true, content: 'ㅠㅠ' },
-	];
+		{ user: true, content: 'ㅠㅠ...' },
+	] );
+
+	// 현재 누구 프로필인지 판단재료
+	const [personNowSending, setPersonNowSending] = useState('true');
+
+	// 입력 받은 메시지를 메시지저장소에 추가
+	function getMessageSubmit(messageText) {
+		if (messageText=='') {
+			alert('메시지부터 입력하세요 ^^');
+			return;
+		}
+
+		setMessageArr([...messageArr, {
+			user: personNowSending,
+			content: messageText
+		}])
+	}
+
+	// 프로필 변경 감지
+	function getHeaderSubmit(person) {
+		setPersonNowSending(!personNowSending);
+	}
```

Comment:
함수명의 시작은 그 함수의 주요**동사**인 것 기억하시죠?
"헤더 제출을 가져온다" 보다는 "메세지 보내고 있는 사람을 바꾼다"가 더 좋아보여용.
그리고 person인자는 인자로 올려보낼 필요가 없는 것 같아요. 안 쓰이는듯??

```suggestion
	function toggleSendingUser() {
		setPersonNowSending(!personNowSending);
	}
```